src/core/dbus-execute.c

```diff
@@ -1151,6 +1152,45 @@ int bus_set_transient_exec_command(
         return 1;
 }
 
+static inline int parse_personality(const char *s, unsigned long *p) {
```

Comment:
I'd drop the "inline". We should use "inline" only in functions implemented in header files. In .c files we can leave it to the compiler to figure out what it wants to inline and what not.